plugins/lando/lando.plugin.zsh

```diff
@@ -0,0 +1,54 @@
+LANDO_ZSH_SITES_DIRECTORY="$HOME/Sites"
+
+LANDO_ZSH_CONFIG_FILE=./.lando.yml
+
+# Enable multiple commands with lando.
+function  artisan \
+          composer \
+          drush \
+          gulp \
+          npm \
+          wp \
+          yarn {
+
+  if checkForFile ; then
+    lando "$0" "$@"
+  else
+    command "$0" "$@"
+  fi
+}
+
+# Check for the file in the current and parent directories.
+checkForFile(){
+
+  local current_directory="$PWD"
+
+  # Bash is backwards. 0 is true 1 (non-zero) is false.
+  # Only bother checking for lando within the Sites directory.
+  if [[ "$PWD" == "$LANDO_ZSH_SITES_DIRECTORY"* ]]; then
```

Comment:
Ok I tested the conditional and here's how it should be written:
```suggestion
  if [[ "$PWD/" == "$LANDO_ZSH_SITES_DIRECTORY"/* ]]; then
```
This makes sure the current working directory is not `$HOME/Sitesandsomethingelse` but still matches the conditional.

Reply:
One more thing: you can flip the logic of the conditional (`!=`) and make an early return. That way you can flatten the rest of the code and it isn't nested inside an if.

Reply:
> One more thing: you can flip the logic of the conditional (`!=`) and make an early return. That way you can flatten the rest of the code and it isn't nested inside an if.

This may not be clear enough. What I mean is:

```
if condition; then
  do something
else
  return 1
fi
```

can be changed to

```
if ! condition; then
  return 1
fi

do something
```

Reply:
Well... i only half understood the early return. I just swapped it but did the following:
```
if ! condition; then
  return 1
else
do something
fi
```
Which, in my defense does the same thing but is unnecessarily verbose. I'll update that.